test/WebJobs.Extensions.Tests/Extensions/DocumentDB/DocumentDBEndToEndTests.cs

```diff
@@ -287,6 +287,7 @@ private Task RunTestAsync(string testName, IDocumentDBServiceFactory factory, Tr
             JobHost host = new JobHost(config);
 
             await host.StartAsync();
+            testTrace.Events.Clear();
```

Comment:
My approach for dealing with the new Warning error messages was just to modify these test helper methods to clear all startup logs before invoking the function. That's what the tests are expecting anyways - they only want invocation logs, not host init/startup logs.

Reply:
Sounds good!